doc/auto-discovery-docker.md

```diff
@@ -1,49 +1,26 @@
 # Set Up Auto Discovery with Docker
 
-This is a guide on how to configure the ONVIF device service for automatic device discovery using Docke.
+This is a guide on how to configure the ONVIF device service for automatic device discovery using Docker.
 
 <img alt="overview" src="images/auto-discovery-docker-overview.jpg" width="75%"/>
 
-Note: For macOS, the network_mode: "host" probably does not work as expected: https://github.com/docker/for-mac/issues/1031
-
-## Non-Security Mode
-
-### Prepare edgex-compose/compose-builder
-
-#### 1. Download the [edgex-compose](https://github.com/edgexfoundry/edgex-compose) and setup it according to the [docker-compose setup guide](./docker-compose/README.md)
-
-#### 2. Update  the `add-device-onvif-camera.yml` file with the following content:
+## Prepare edgex-compose/compose-builder
+1. Download the [edgex-compose](https://github.com/edgexfoundry/edgex-compose)
+2. Change the directory to ` edgex-compose/compose-builder`
 
+## Update the `./compose-builder/add-device-onvif-camera.yml`
```

Comment:
probably don't need the `./compose-builder` part since the command above you have them cd into it.

Reply:
Removed the `./compose-builder` part .